bucket/opennetmeter.json

```
@@ -0,0 +1,19 @@
+{
+    "version": "0.10.0",
+    "description": "A simple program to monitor your network/data usage.",
+    "homepage": "https://github.com/Ashfaaq18/OpenNetMeter",
+    "license": "Apache-2.0",
+    "suggest": "windowsdesktop-runtime-lts",

```

```suggestion
    "suggest": {
        "Windows Desktop Runtime": "extras/windowsdesktop-runtime-lts"
    },
```

`suggest` should be a *dictionary*.

It can work without specifying the bucket (e.g. `"Windows Desktop Runtime": "windowsdesktop-runtime-lts"`), it is just a convention to specify bucket as well.